src/main/java/nextstep/ladder/domain/LineFactory.java

```diff
@@ -13,28 +15,73 @@ private LineFactory() {
     private static Random random = new Random();
 
     public static Line createLine(int countOfPerson) {
-        List<Boolean> points = randomPoints(countOfPerson);
-        validatePoints(points);
-        return new Line(points);
+        if (countOfPerson == 1) {
+            return new Line(List.of(STAY));
```

Comment:
```suggestion
            return new Line(STAY);
```

생성자를 다양하게 활용해볼것 같아요!